src/main/java/org/prebid/server/metric/Metrics.java

```diff
@@ -149,20 +155,28 @@ public void updateAdapterRequestErrorMetric(String bidder, MetricName errorMetri
         forAdapter(bidder).request().incCounter(errorMetric);
     }
 
-    public void updateCookieSyncRequestMetric() {
+    public void updateUserSyncRequestMetric() {
```

Comment:
Seems this should remain as `updateCookieSyncRequestMetric(...)` since it updates common `cookie_sync_requests` metric.